### ISSUE TITLE
BUG: torch.symeig

### ISSUE THREAD

**Issue by Molugan** [facebookresearch/pytorch_GAN_zoo#61]
When GDPP is active, torch.symeig may generate NaN values depending on the random seed used for training.
